### PR TITLE
LST-1160 Phase one of adding group/business unit: Add models and admin.

### PR DESCRIPTION
**Pull request by oskusalerma**
This is needed so we can setup the data ahead of time before rolling out
the actual functionality using the data.

### CODE REVIEW

File: main/models.py [hunk 1]
```diff
@@ -1,7 +1,41 @@
 from django.conf import settings
+from django.core.exceptions import ValidationError
 from django.db import models
 
 
+def validate_business_units(value):
+    business_units = [x.strip() for x in value.strip().split("\n") if x.strip()]
```
Comment (rossmiller) [inline main/models.py:7]: Do you need some error handling here? If value is `None`, you will get `AttributeError: 'NoneType' object has no attribute 'strip' `

File: main/models.py [hunk 2]
```diff
@@ -1,7 +1,41 @@
...
+        help_text="One business unit per line", validators=[validate_business_units]
+    )
+
+    def __str__(self):
+        return self.name
+
+    def get_business_units(self):
+        """Get a list of business units for this group."""
+
+        business_units = [x.strip() for x in self.business_units.strip().split("\n") if x.strip()]
```
Comment (rossmiller) [inline main/models.py:30]: As above.

**Review by rossmiller — APPROVED** [review]
LGTM apart from some error handling that MIGHT be needed.

File: main/models.py (hunk 1 above)
Comment (oskusalerma) [reply main/models.py]: done

**Pull request merged**